### PR TITLE
Gitlab tracker: Duplicate issues

### PR DESCRIPTION
**Pull request by skahn007gl**
## Proposed changes

Small update to Gitlab tracker to remove duplicate issue creation for duplicate findings.
Instead of raising a new issue, the newest existing issue updated with a comment containing the new findings and re-opened if required.

## Checklist

- [X] Pull request is created against the [dev](https://github.com/projectdiscovery/nuclei/tree/dev) branch
- [X] All checks passed (lint, unit/integration/regression tests etc.) with my changes
- [X] I have added tests that prove my fix is effective or that my feature works
- [X] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Review by dogancanbakir — CHANGES_REQUESTED** [review]
LGTM -left a comment for doc update.

**Comment by dogancanbakir** [comment]
Forgot to add `duplicate-issue-check: true` in config. Checking it again.. 🤦

File: docs/getting-started/running.mdx [hunk 1]
```diff
@@ -671,7 +671,7 @@ Nuclei comes with reporting module support with the release of [v2.3.0](https://
...
...                                                                                                                                                                                                                                                                                   | SARIF                                                                                                                                                                                                                                                                                                             | Elasticsearch                                                                                                                                                                                                                                                                                                     | Splunk HEC                                                                                                                                                                                                                                                                                                        |
```
Comment (dogancanbakir) [inline docs/getting-started/running.mdx:672]: Could you please include the `duplicate-issue-check: true` in the "Nuclei Reporting" section as well?

**Review by dogancanbakir — APPROVED** [review]
LGTM

**Review by ehsandeep — APPROVED** [review]

**Pull request merged**